src/App.tsx

```diff
@@ -28,6 +30,14 @@ function App() {
       <p className="read-the-docs">
         Click on the Vite and React logos to learn more
       </p>
+      <Button onClick={() => {
```

Comment:
Tohle by na masteru idealne nemelo byt. Mela bys idealne mergovat jen realnej kod (ale vim, celej ten app soubor je stejne blbost :)

Reply:
To tak nějak sem to brala že tam ještě nemáme to co tam bude, ale ok, beru na vědomí. Pro mě je ale třeba někdy těžký rozklíčovat jestli mám chybu přímo v komponentě nebo v tom jak je použitá..

Reply:
tím pádem až budeme posílat na pul request, tak jen addneme, commitneme a pushneme  ten konkrétní soubor. `git add nazev-souboru`?

Reply:
Jde o to pridat do commitu jen to co ma realne byt na mastru. Testovat si to muzete kde chcete a nacem chcete. Ale pokud to realne nema pouziti a neni ta zmena zadana, nemelo by to tam co delat

Reply:
přidala sem na to na discord vlákno vykresleni-komponenty tak to můžeme dávat tam at to každej nemusí na code review vymýšlet zvlášť..